Automattic/jetpack
# Update monorepo to Composer 2.6

## Proposed changes:

Composer 2.6 has been out for a while now, and Renovate is starting to generate diffs that want to bump the version number in the lockfile. Time for us to upgrade.

### Other information:


## Jetpack product discussion


None
Announcement draft: pdWQjU-zH-p2

## Does this pull request change what data or activity we track or use?


No

## Testing instructions:






* If you don't update composer locally, `jetpack` commands should give an error. Once you update, the error should go away.
